clients/java/src/main/java/io/camunda/client/CamundaClientBuilder.java

```diff
@@ -145,6 +145,12 @@ CamundaClientBuilder jobWorkerExecutor(
   /** The request timeout used if not overridden by the command. Default is 10 seconds. */
   CamundaClientBuilder defaultRequestTimeout(Duration requestTimeout);
 
+  /**
+   * The {@link io.camunda.client.impl.command.ActivateJobsCommandImpl}'s response timeout offset.
+   * Default is 1 second.
+   */
+  CamundaClientBuilder defaultActivateJobsResponseTimeoutOffset(Duration responseTimeoutOffset);
```

Comment:
🔧 To align it with the above, I guess it should be **request** timeout, not response timeout?

Reply:
Hm, hard to say, actually. We set the response timeout in the HTTP request, not the request timeout. For the client user it might be confusing though to use response timeout here now.

Reply:
Yes, I was about to say that this config is used in the HTTP response timeout. So it's the client side timeout.

Reply:
But yeah, the `defaultRequestTimeout` is also used in HTTP response timeout for all  commands. I guess then it is confusing, I'll just call it requestTimeout, too.

Reply:
I'm thinking about `defaultRequestTimeoutClientOffset`.

Reply:
But we are in the client builder, so we're setting the request timeout offset for the client already, aren't we? 😆

Reply:
Well yes, but we also use the `defaultRequestTimeout` on the server.
Idk..